packages/@azure/servicebus/data-plane/lib/queueClient.ts

```diff
@@ -133,11 +130,11 @@ export class QueueClient extends Client {
 
   /**
    * Sends a batch of SendableMessageInfo to the ServiceBus Queue in a single AMQP message.
-   * - For sending a message to a `session` enabled Queue, please set the `sessionId` property of
-   * the message.
-   * - For sending a message to a `partition` enabled Queue, please set the `partitionKey` property
-   * of the message.
-   * For more information please see {@link https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-partitioning#use-of-partition-keys Use of partition keys}
+   * To send messages to a `session` or `partition` enabled Queue, please set the
+   * `sessionId` property and `partitionKey` properties respectively.
+   *
+   * All messages in the batch should have the same `sessionId` and `partitionKey` properties.
```

Comment:
> All messages in the batch should have the same `sessionId` and `partitionKey` properties.

Only for sending to partition enabled entities, right ?

Reply:
Or even session enabled ones.

The idea is if user provides either sessionId or partitionKey, they should be same across all the messages in the batch

Reply:
Okay, I wanted to mention that 
"for sending to non-partition enabled entities, we don't need all messages in the batch to have same sessionId and partitionKey properties."


> All messages in the batch should have the same `sessionId` and `partitionKey` properties.

This conveys that - 'sendBatch() won't work if we don't have same `sessionId`/`partitionKey` properties for all the messages'. (which is not true for sending to Unpartitioned entities.)

Reply:
Ok, I can make that clearer